docs/charts/doughnut.md

```diff
@@ -14,26 +14,26 @@ They are also registered under two aliases in the `Chart` core. Other than their
         "labels": [
             "Red",
             "Blue",
-            "Yellow",
+            "Yellow"
```

Comment:
Should we use single quotes on the right and no quotes on the left hand side?

Reply:
I would prefer not, it looks pretty bad and not consistent with our source code:

```javascript
"labels": [
   "Red"
   , "Blue"
   , "Yellow"
]
```

Reply:
I don't understand your response, so maybe my proposal wasn't clear. What I was suggesting was to do the following, which I think is more consistent with the syntax we use elsewhere:

```
labels: [
    'Red',
    'Blue',
    'Yellow'
]
```

Reply:
I implicitly associated your comment to the code change (mixed "quote" with "comma").

+1 for single quote, when needed

Reply:
This is the embedded js code, so the changes will not impact the appearance. But I agree that it is always good to make the styles cosistent.

Reply:
And I think that's json, so double quotes required.

Reply:
Ah, that makes sense.

Reply:
I'm not sure that it is JSON. It generates the following, which I think should just be javascript and not json:

```
<script>new Chart(document.getElementById("chartjs-4"),{"type":"doughnut","data":{"labels":["Red","Blue","Yellow"],"datasets":[{"label":"My First Dataset","data":[300,50,100],"backgroundColor":["rgb(255, 99, 132)","rgb(54, 162, 235)","rgb(255, 205, 86)"]}]}});</script>
```

Reply:
https://github.com/chartjs/gitbook-plugin-chartjs/blob/master/index.js#L12